components/caseStudyDetails/BulletPoints.tsx

```diff
@@ -0,0 +1,48 @@
+"use client";
+
+import { motion } from "framer-motion";
+import Image from "next/image";
+
+import { generateFadeInAnimationVariants } from "@/utils";
+
+const BulletPoints = ({
+  bulletPoints,
+  challengesOrLearnings,
+  icon,
```

Comment:
As you have previously done in the code, could you place it in a separate file with similar code?